<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeAndDockClass</class>
 <widget class="QMainWindow" name="TreeAndDockClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TreeAndDock</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>34</y>
      <width>611</width>
      <height>371</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>609</width>
       <height>369</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="actAddFolder"/>
    <addaction name="actAddFiles"/>
    <addaction name="actDeleteItem"/>
    <addaction name="actScanItems"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actZoomIn"/>
    <addaction name="actZoomOut"/>
    <addaction name="actZoomRealSize"/>
    <addaction name="actZoomFitW"/>
    <addaction name="actZoomFitH"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddFiles"/>
   <addaction name="actDeleteItem"/>
   <addaction name="actScanItems"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomRealSize"/>
   <addaction name="actZoomFitW"/>
   <addaction name="actZoomFitH"/>
   <addaction name="separator"/>
   <addaction name="actDockFloat"/>
   <addaction name="actDockVisible"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>112</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>图片目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>27</number>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>节点</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddFolder">
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/open3.bmp</normaloff>:/images/icons/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>添加目录...</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/824.bmp</normaloff>:/images/icons/824.bmp</iconset>
   </property>
   <property name="text">
    <string>添加文件...</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/418.bmp</normaloff>:/images/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/416.bmp</normaloff>:/images/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/414.bmp</normaloff>:/images/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="toolTip">
    <string>图片实际大小显示</string>
   </property>
  </action>
  <action name="actZoomFitW">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/424.bmp</normaloff>:/images/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="toolTip">
    <string>适合宽度显示</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actDeleteItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/delete1.bmp</normaloff>:/images/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/exit.bmp</normaloff>:/images/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actZoomFitH">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/426.bmp</normaloff>:/images/icons/426.bmp</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actScanItems">
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/fold.bmp</normaloff>:/images/icons/fold.bmp</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/414.bmp</normaloff>:/images/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
   <property name="toolTip">
    <string>窗体可见</string>
   </property>
  </action>
  <action name="actDockFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TreeAndDock.qrc">
     <normaloff>:/images/icons/814.bmp</normaloff>:/images/icons/814.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
   <property name="toolTip">
    <string>窗体浮动</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TreeAndDock.qrc"/>
  <include location="TreeAndDock.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>TreeAndDockClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
